### DISCUSSION TITLE
Viewing cached packages in web UI?

### DISCUSSION THREAD

**Discussion by Artalus** [devpi/devpi#881] [Q&A] [answered]
Hi!

We have a `devpi-server` with `devpi-web` set up as a caching proxy for Pypi. Is it possible to see which exact packages and files were cached for `root/pypi` index? The web UI seemingly provides only the whole (350k+) list of packages in upstream Pypi.

**Comment by fschulze** [comment, marked as answer]
This is currently not possible, I created an issue for it: https://github.com/devpi/devpi/issues/882
